### ISSUE TITLE
Autostyling on category widget should respect the size of the points and change just the color.

### ISSUE THREAD

**Issue by saleiva** [CartoDB/deep-insights.js#341]

**Comment by saleiva** [comment]
hey guys, after testing this is quite important since in some cases the points are very hard to visualize

**Comment by saleiva** [comment]
cc @javisantana

**Comment by fdansv** [comment]
The solution is not trivial though, as this means we have to read individual properties from the current state of the layer's style.

**Comment by saleiva** [comment]
yeah dude, but this invalidates the auto-styling in most of the cases.

2016-06-14 13:19 GMT+02:00 Francisco Dans <notifications@github.com>:

> The solution is not trivial though, as this means we have to read
> individual properties from the current state of the layer's style.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CartoDB/deep-insights.js/issues/341#issuecomment-225851928>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAIENfNd8tmpFSfO0NhvoCEj4poSQ3kGks5qLo5MgaJpZM4IYBIR>
> .
>



-- 
Sergio Alvarez Leiva

**Comment by javisantana** [comment]
We left out this in the first stage in order to speed up the release because it's not that easy but it's not actually hard.

As we have the carto parser and we know the properties we are going to change is easy to get the important ones (for example, `marker-width`) and respect it when generating the new cartocss.

This is an example of how to get a specific property: https://github.com/CartoDB/cartodb.js/blob/af21316c44894f67341c65453eba97254effdeda/src/geo/map/torque-layer.js#L68
